app.js

```diff
@@ -1,9 +1,14 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./controllers/topic-controller");
+const { getTopics } = require("./controllers/controllers");
+const endpoints = require("./endpoints.json");
```

Comment:
would be nice to require in endpoints in the controller and send.

Reply:
Hi Nick, I'm not 100% sure what you mean!